src/main/java/org/cactoos/text/AbbreviatedText.java

```diff
@@ -0,0 +1,104 @@
+/**
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2017-2018 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package org.cactoos.text;
+
+import java.io.IOException;
+import org.cactoos.Text;
+
+/**
+ * Abbreviates a Text using ellipses.
+ *
+ * <p>There is no thread-safety guarantee.
+ *
+ * @author Fabricio Cabral (fabriciofx@gmail.com)
+ * @version $Id$
+ * @since 0.29
+ */
+public final class AbbreviatedText implements Text {
+
+    /**
+     * The origin Text.
+     */
+    private final Text origin;
+
+    /**
+     * The max width of the resulting string.
+     */
+    private final int width;
+
+    /**
+     * Ctor.
+     * @param text The Text
+     */
+    public AbbreviatedText(final String text) {
+        this(new TextOf(text));
+    }
+
+    /**
+     * Ctor.
+     * @param text The Text
+     */
+    public AbbreviatedText(final Text text) {
+        // @checkstyle MagicNumber (1 line)
+        this(text, 80);
```

Comment:
@fabriciofx This 80 could be taken into a private constant, same as the 3 below. I would name them ``MAX_LENGTH`` and ``ELLIPSES``. Are private constants forbidden by Qulice? I don't remember. At least the constant ``ELLIPSES`` would be nice because otherwise it is not so clear why ``this.width - 3`` in the ``asString()`` method

Reply:
@amihaiemil I agree. I'll change it. Thanks!